react/features/chat/middleware.js

```diff
@@ -264,7 +280,7 @@ function _addChatMsgListener(conference, store) {
                     batch(() => {
                         store.dispatch(setToolboxVisible(true));
                         store.dispatch(setToolboxTimeout(
-                                () => store.dispatch(hideToolbox()),
+                                () => store.dispatch(hideToolbox ? hideToolbox() : setToolboxVisible(false)),
```

Comment:
Why do we want to show / hide the toolbox when a reaction is received?

Reply:
Because the animated emojis start from the toolbox

Reply:
I don't think that's a good idea. How about Jibri?

Reply:
I don't think we want them on recordings. Not sure though

Reply:
As discussed, we can remove this now. The chat notification issue is separate.